packages/contracts/test/voting/token-voting.ts

```diff
@@ -509,19 +551,17 @@ describe('TokenVoting', function () {
   });
 
   describe('Configurations for different use cases', async () => {
-    const id = 0; // proposalId
-
-    describe('A simple majority vote with >50% relative support and >25% total support required', async () => {
-      let minDuration = 500;
-      let relativeSupportThresholdPct = pct16(50);
-      let totalSupportThresholdPct = pct16(25);
-      let totalVotingPower = 100;
+    describe('A simple majority vote with >50% support and >25% participation required', async () => {
+      supportThreshold = pct16(50);
+      participationThreshold = pct16(25);
```

Comment:
Is `pct16` a way of encoding ratios?

Reply:
Yes, we do 
```
const toBn = ethers.BigNumber.from;
const bigExp = (x: number, y: number) => toBn(x).mul(toBn(10).pow(toBn(y)));
export const pct16 = (x: number) => bigExp(x, 16);
```
Let's tackle this in https://aragonassociation.atlassian.net/browse/APP-1264